{{ cookiecutter.repo_name }}/docker/action/README.md

```diff
@@ -0,0 +1,25 @@
+# Dockerfile CI action
+
+This action executes arbitrary commands within a Docker container defined by
+the local Dockerfile. Commands are passed to the container via the `command`
+input and received by an entrypoint script `action-exec.sh` that feeds the
+command string to `bash -c`
```

Comment:
Can you put a link to the GitHub actions documentation that covers this?

Reply:
Added the following to the action README:
## Documentation
The documentation for how this action works can be found here:
* [Creating a Docker container action](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-docker-container-action)
* [Dockerfile support for GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/dockerfile-support-for-github-actions)